### PR TITLE
[BRAN-393] Mosaic Release Notification

### PR DESCRIPTION
**Pull request by Alderr**
## Purpose/Description:

- After testing a million times, added the ability to notify our Slack when Mosaic is updated to a new version

## Jira Link:

[BRAN-393](https://collagegroup.atlassian.net/browse/BRAN-393?atlOrigin=eyJpIjoiZWYxMjRiMDQ1ZjUzNDQ3ZGI5NDZmODJjMDgzMGIwOTIiLCJwIjoiaiJ9)

[BRAN-393]: https://collagegroup.atlassian.net/browse/BRAN-393?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by diego20 — APPROVED** [review]
LGTM.
+1 for the accurate description

**Review by darkares23 — APPROVED** [review]
LGTM, we should do the same in the BE

File: .github/workflows/new-version-notification.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+          else
+            echo "header=Merged PRs:" >> $GITHUB_ENV
+          fi
+
+      - name: Send Slack Notification
+        uses: slackapi/slack-github-action@v1
+        with:
+          channel-id: ${{ secrets.SLACK_CHANNEL_ID }}
+          slack-message: |
+            <https://646227b52910551bc013d3ef-rplhylrpxw.chromatic.com|Successfully released mosaic version ${{ env. current_tag }} :books:.>
```
Comment (baugusli) [inline .github/workflows/new-version-notification.yml]: There's an extra space for `env. current_tag` I think that would cause an issue? I didn't test it

**Review by baugusli — COMMENTED** [review]
Just one comment on the extra space for `env. current_tag` which should be `env.current_tag` ideally. I didn't test this though

File: .github/workflows/new-version-notification.yml (hunk 1 above)
Comment (Alderr) [reply .github/workflows/new-version-notification.yml]: ah nice catch, addressing

**Pull request merged**